strapi/strapi
# Make the BoundRoute section sticky in the Roles editor

### What does it do?
This PR is supposed making the `BoundRoute` section sticky in the Roles editor but another solution has been found out.
fixes #12300

### Why is it needed?
we have found out that  making the `BoundRoute` section position sticky was not really important at all since `BoundRoute` section can not exceed more than five `BoundRoutes` at the time. We've ended up by making the `UserPermission` section `scrollable` instead and fixing the height.
### Describe the issue you are solving.
We are giving user ability of having a look at `BoundRoutes` on BoundRoute section when he is selecting roles, instead of scrolling to the top all the time when he wants to see `BoundRoutes` selected. Video demo will describe it clearly.
[video](https://www.loom.com/share/b21e6791ef1d418e94fbcb5b3cbf8dc5)
### How to test it?
-  make sure you are in the project folder
-  execute  `cd ./packages/core/admin` and run the server by typing `yarn develop`
-  changes made on the Dashboard can be accessed the this link  this `http://localhost:4000/admin/settings/users-permissions/roles`
-  [video demo](https://www.loom.com/share/1d2217ab14844b7694b526cccaa71b5c) 

---
This code was written and reviewed by GitStart Community. Growing future engineers, one PR at a time.
